src/domain/communication/discussion/providers/forum-queries.graphql

```diff
@@ -46,6 +46,14 @@ query platformDiscussions {
         comments {
           id
           messagesCount
+          messages {
```

Comment:
This looks dangerous, as it will result in loading all messages from all rooms on the forum I think?

Reply:
Yes, this was previously commentsCount but now it became Comments. I am adjusting the Room interface on the client side to have messages optionally.